diff -Nur cairo/test/make-cairo-test-constructors.py cairo-test-constructors/test/make-cairo-test-constructors.py
--- cairo/test/make-cairo-test-constructors.py	2018-11-25 19:00:08.685865741 -0800
+++ cairo-test-constructors/test/make-cairo-test-constructors.py	2018-11-25 19:04:50.510467136 -0800
@@ -15,6 +15,9 @@
 
     for fname in args.input:
         with open(fname, 'r') as f:
+            print("------------------------------make-cairo-test-constructors-1")
+            print(fname)
+            print("------------------------------make-cairo-test-constructors-2")
             for l in f.readlines():
                 if match_test_line.match(l):
                     test_names.append(match_test_name.match(l).group(1))
